src/EnumStringEditor.tsx

```diff
@@ -1,19 +1,21 @@
 import { Radio } from "antd";
 import { Field, FormikProps } from "formik";
 import * as React from "react";
+import { RadioGroupProps } from "antd/lib/radio/interface";
+import { FormikFieldProps, iDataSourceObject } from "./FieldProps";
 
-export const EnumStringEditor = (props: any) => (
-  <Field {...props}>
-    {({ field, form }: { field: any; form: FormikProps<any> }) => {
-      return (
-        <Radio.Group {...field}>
-          {props.dataSource.map((item: any) => (
-            <Radio key={item.value} value={item.value}>
-              {item.displayName}
+export const EnumStringEditor = (
+  { name, validate, dataSource, ...restProps }: FormikFieldProps & RadioGroupProps & { dataSource: Array<iDataSourceObject> }
+) => (
+  <Field name={name} validate={validate}>
+    {({ field, form }: { field: any; form: FormikProps<any> }) => (
+        <Radio.Group {...restProps}>
+          {dataSource.map(({ value, label }: iDataSourceObject ) => (
+            <Radio key={value} value={value}>
+              {label}
```

Comment:
Imho displayName should be used instead of label (as it was before).
Label (as the label prop of FieldItem) is a static value in the UI indicating a field ("firstname" or "lastname"). DisplayName here means the user-friendly value of an complex object. Imagine a user object { displayName: "Peter", value/id: 123 }
So displayName and label are two different things, and it should be distinguished in the code

Reply:
To me displayName and label mean the same thing, I felt that `label` had a better convention associated to it, but I'm willing to concede this naming convention.

If we were to change `label` to `displayName`, may I suggest following this convention for future components that will have an added `dataSource` like #11  and future SelectField PR that will add similar functionality

I will add a commit to revert this change.

Reply:
yes, pls revert to displayName. For future components we will also use this convention.
When I design backend apis, I also take this into account, a typical "option" object at least has a displayName and value (or id) field (at least if there is no other argument against it).
Label in general is also a good name, but it clashes with a static UI label thingy.

Reply:
It seems your choice of 'label' adheres more to how it is done in antd. The RadioGroup documentation about the options property takes an Array<{label:string, value: string, disabled?:boolean}>. If possible we can expose this through the api (instead of IDataSourceObject).
PLS leave the the EnumStringEditor.tsx (an experimental component anyway) as suggested in your PR